arm-image-installer

```diff
@@ -606,6 +632,34 @@ if [ "$SSH_KEY" != "" ]; then
 		echo "= WARNING: No SSH Key Added."
 	fi
 fi
+# Configure Wi-Fi if SSID and password are provided
+if [ "$WIFI_SSID" != "" ] && [ "$WIFI_PASS" != "" ]; then
+    echo "= Configuring Wi-Fi: $WIFI_SSID"
+    mkdir -p ${PREFIX}/etc/NetworkManager/system-connections/
+    # Use the SSID as the filename (replace spaces and special characters with underscores)
+    FILENAME=$(echo "$WIFI_SSID" | tr ' ' '_' | tr -cd '[:alnum:]._-').nmconnection
+    cat > ${PREFIX}/etc/NetworkManager/system-connections/$FILENAME <<-EOF
+[connection]
+id=$WIFI_SSID
+type=wifi
+interface-name=wlan0
```

Comment:
We can't assume that is the case. In different systems I have wlP6p1s0, wlan0, wlp1s0, wlp4s0, wlu1 and others.

Reply:
removed the parameter, let networkmanager handle it.

Reply:
Does it work without the parameter though?